zvision_lidar_node:
    ros__parameters:
        device_ip: 192.168.10.108
        udp_port: 2368
        read_once: false
        read_fast: false
        frame_id: zvision_lidar
        model: ML30SA1
        frequency: 10
        pcap: ""
        time_offset: 0.0
